rest/pagination/README.md

```diff
@@ -0,0 +1,78 @@
+# Pagination with `@rest`
+
+This snippet shows how `@rest` fields are configured to support pagination.
```

Comment:
Is there a better description than `@rest` field?   Can we provide a link to our documentation or a snippet for @rest?

"how fields defined with StepZen's @rest directive" maybe?